src/components/organisms/UiMultipleChoices/UiMultipleChoices.scss

```diff
@@ -0,0 +1,12 @@
+
+//<Code id="multiple-choice-side-panel">
+.multiple-choices-side-panel {
+  &__heading {
+    margin-top: var(--space-32);
+    margin-bottom: var(--space-16)
+  }
+  .ui-text--body-1-thick {
```

Comment:
WHAT?! it doesn't make sense. You shouldn't use our class to do that. You should add a custom class to this element and use variable instead static color

Reply:
Sorry 😢